test/pkg.jl

```diff
@@ -287,72 +264,46 @@ temp_pkg_dir() do
     end
 
     # Various pin/free/re-pin/change-pin patterns (issue #17176)
-    begin
-        ret, out, err = @grab_outputs Pkg.free("Example")
-        @test ret === nothing && out == ""
-        @test contains(err, "INFO: Freeing Example")
-
-        ret, out, err = @grab_outputs Pkg.pin("Example")
-        @test ret === nothing && out == ""
-        @test ismatch(r"INFO: Creating Example branch pinned\.[0-9a-f]{8}\.tmp", err)
-        @test !contains(err, "INFO: No packages to install, update or remove")
```

Comment:
It wasn't deleted, and in fact the test was strengthened:  The new regex test has `r"^...$"` so that it checks that exactly one `INFO` message is printed.